packages/phrases/src/locales/de/translation/oidc.ts

```diff
@@ -0,0 +1,5 @@
+const oidc = {
+  logout_success: 'You have been successfully signed out.',
```

Comment:
nit: maybe add `// UNTRANSLATED` tags for all non-English languages so that we can simply locate them